Source/automap.cpp

```diff
@@ -230,22 +230,22 @@ void DrawAutomapTile(CelOutputBuffer out, Sint32 sx, Sint32 sy, Uint16 automap_t
 	}
 }
 
-void DrawAutomapItem(CelOutputBuffer out, Sint32 x, Sint32 y, Uint8 color)
+void DrawAutomapItem(const CelOutputBuffer &out, int x, int y, uint8_t color)
 {
-	Sint32 x1 = x - AmLine32 / 2;
-	Sint32 y1 = y - AmLine16 / 2;
-	Sint32 x2 = x1 + AmLine64 / 2;
-	Sint32 y2 = y1 + AmLine32 / 2;

```

Comment:
When do we use int and when Sint32?

Reply:
Generally `int` unless the exact width is important

Reply:
`int`: when 16 bit might be is enough and 32 bit fore sure is enough, and 64 bit is defiantly enough.

`Sint32`: Only if we are eating some lower part of SDL that would rather prefer we spoke it's dialect.

int32_t: when only 32bit will do.